charts/reform-scan-notification-service/values.yaml

```diff
@@ -21,6 +21,7 @@ java:
     DB_CONN_OPTIONS: ?sslmode=require
     FLYWAY_SKIP_MIGRATIONS: true
     QUEUE_NOTIFICATIONS_MAX_RETRY: 5
+    S2S_URL: http://rpe-service-auth-provider-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
```

Comment:
need to override the preview with aat value ;)

Reply:
good catch

Reply:
There is no S2S_URL in preview.template.yaml in processor

Reply:
that's what we are saying - need to override (include into preview.values) to use aat value